felles/log/src/main/java/no/nav/vedtak/log/audit/CefFields.java

```diff
@@ -0,0 +1,18 @@
+package no.nav.vedtak.log.audit;
+
+public class CefFields {
```

Comment:
kunne vært enum?  trenger feltene være public konstanter?

Reply:
Dette er gjort for å støtte utvidelser per applikasjon siden kun enkelte headere har blitt definert. Jeg har vurdert en uttømmende liste med alle mulige CEF-headere (stor enum). Diskuterer gjerne dette litt mer på Slack.

Reply:
ok